tests/test_backend.py

```diff
@@ -1,3 +1,5 @@
+# -*- coding: utf-8 -*-
```

Comment:
This is required to prevent syntax errors when importing the test file, now that it contains unicode chars.